packages/server/__tests__/api/grants.test.js

```diff
@@ -324,6 +324,19 @@ describe('`/api/grants` endpoint', () => {
             });
         });
     });
+    context('DELETE /api/grants/:grantId/interested/:agencyId', () => {
+        context('by an admin user', () => {
+            const interestEndpoint = `335255/interested`;
+            it('records this admin\'s ability to delete a grant', async () => {
+                const response = await fetchApi(`/grants/${interestEndpoint}/undefined`, agencies.own, {
+                    ...fetchOptions.admin,
+                    method: 'delete',
+                    body: JSON.stringify({ agencyIds: [agencies.own], interestedCode: null }),
```

Comment:
Can you add tests that:
1. Attempt to unmark a grant as interested from multiple agencies (although nothing does this now, the API theoretically supports it, so we should test it)
2. Attempt to unmark a grant as interested when at least one of the specified agencies is of a different tenant (should fail)

Reply:
Done ✅